src/otaclient/client_package.py

```diff
@@ -486,7 +475,7 @@ def mount_client_package(self) -> None:
             self._create_mount_namespaces()
             self._mount_squashfs_file()
             self._bind_mount_host_dirs()
-            self._bind_mount_active_slot()
+            self._prepare_active_and_standby_slots()
```

Comment:
In the client mount sequence, erase standby then mount both standby and active slots.